scripts/enum/animation_strings.lua

```diff
@@ -0,0 +1,107 @@
+-----------------------------------
+-- Animation strings
+-----------------------------------
+xi = xi or {}
+
+--[[
+Looks like in other places we call them entityAnimation and in another for zone parts using a diff packet but the same index (like lamps in psoxja, geysers in dangruf wadi) uses entityVisual
+
+-----------------------------------
+mob:entityAnimationPacket(xi.animationString.STATUS_VISIBLE)
+-----------------------------------
+The base entity is a mob, and anyone in range to see the mob sees the animation.
+We do accept a second argument in this one, for target.
+An example of this is the afflictors in Beadeaux, which target the player. npc:entityAnimationPacket('main', player)
+
+-----------------------------------
+player:entityVisualPacket('saa1')
+-----------------------------------
+In this there is no mob  or npc at all.
+Tt's a part of the zone itself animating, so player is used as base entity and any player that zones in gets sent the packet to light up that part of the zone.
+So the thing animating it's really the zone itself, sort of.
+(There is technically "an object"  or "entity" but its not ID'd like an NPC, and technically speaking a zone dat is laid out like NPCs are part of it anyway)
+
+-----------------------------------
+SendEntityVisualPacket(ID.npc.GEYSER_OFFSET, 'kkj2')
+-----------------------------------
+This one is a luautils binding, sending a packet to the entire zone (of whoever is in visual range of the ID specified) on demand.
+
+This are FOURCC codes.
+For most things you can get away with using the wrong one, and they still just work.
+--]]
+
+xi.animationString =
```

Comment:
Seems reasonable to me.  My only question is should we nest tables on this with these logical groupings instead of having them all in one?

Reply:
I think we can kick the grouping idea down the road for now, because if we group we're going to have to settle on how we should group them and right now I don't see a way to do so that we won't regret later. These are "kind of sort of" model specific, yet multiple models can have the same one. And some are part of the zone instead of a specific npc id. (Or a mob in rarer cases that are usually a hacky work around.) One big list is the lesser evil at this time.